packages/addons-inject/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:18.15
+COPY package.json /usr/src/app/packages/addons-inject/
+COPY index.js /usr/src/app/packages/addons-inject/
+COPY wrangler.toml /usr/src/app/packages/addons-inject/
```

Comment:
Shouldn't this be volume?

Reply:
It's mounted as a volume in our docker compose file, and this volume will override this path. This Dockerfile was previously was just installing a hardcoded version of wrangler, the copy operations are just for the initial build.